### ISSUE TITLE
Fill in the blanks question not displayed properly

### ISSUE THREAD

**Issue by r-torres-csps** [CSPS-RIW/Vue_QuizGen-riw-patches#48]
Filli n the blanks component not rendering proper data or lacking enough data generated by [quizgen creator](https://quizgencreator.netlify.app/quiz-manager)
